snippets/accessibility/AccessibleLinks.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Snippet Title: Accessible Link Colors
+ * Scope:         global
+ * Goal:          accessibility
+ * Description:   Makes most links across the site blue and underlined, and purple if visited. Includes activity links, but excludes section titles, buttons and other link controls across the site
+ * Creator:       Pierre Camaly de Brosses
+ * Tracker issue: <snippet tracker issue>

```

Comment:
You can omit this line if your snippet is not targetted to any Moodle core tracker issue.